02-peer-protocol.md

```diff
@@ -968,6 +997,42 @@ additions.
 
 The accepting node:
     - MAY respond with a `funding_satoshis` value of zero.
+    - if the `option_will_fund` tlv was sent in `open_channel2`:
+        - if they decline to contribute funds or to be paid for their
+          contributions:
+            - SHOULD omit the `will_fund` tlv
+        - if they decide to accept the offer:
+            - MUST include a `will_fund` tlv
+            - MUST set `funding_satoshis` to a value greater than 0msat
+            - MAY provide a `funding_satoshis` of any amount.
```

Comment:
Does this imply WUMBO?

Reply:
Not intentionally; I mean to signal that you MAY provide less (or more) than `requested_sats`.

Will update to `MAY send less`...